verifier/src/lib.rs

```diff
@@ -13,6 +14,8 @@ pub use report_body::{
     MiscellaneousSelectVerifier, MrEnclaveVerifier, MrSignerVerifier, ReportDataVerifier,
 };
 
+pub use certs::{Error as CertificateError, UnverifiedCertificate, VerifiedCertificate};
```

Comment:
These are exported here to make cargo happy. Current htinking is that these will be hid behind
`CertificationData::verify` or similar